cloudify_cli/config/parser_config.py

```diff
@@ -244,26 +247,27 @@ def auto_generate_ids_argument():
 
 def parser_config():
     return {
-        'description': 'Manages Cloudify in different Cloud Environments',
+        'description': "Cloudify's Command Line Interface",
         'arguments': {
             '--version': {
-                'help': 'Show version information and exit',
+                'help': 'show version information and exit',
                 'action': cfy.version
             }
         },
         'commands': {
             'logs': {
-                'help': 'Handle Cloudify Manager logs',
+                'help': "Handle the Manager's logs",
                 'sub_commands': {
                     'get': {
```

Comment:
let's forget the implementation for a second and concentrate on the help texts. If it bothers you, please open a JIRA issue for it.